domain/removal/state/model/model.go

```diff
@@ -483,13 +417,19 @@ WHERE  model_uuid = $entityUUID.uuid;`, model, modelUUID)
 	return life.Life(model.Life), nil
 }
 
-func (st *State) checkNoModelDependents(ctx context.Context, tx *sqlair.TX) error {
+func (st *State) checkNoModelDependents(ctx context.Context, tx *sqlair.TX, force bool) error {
+	// If we're forcing, we don't care about dependents.
+	if force {
+		st.logger.Infof(ctx, "force flag set, skipping model dependents check")
+		return nil
+	}
```

Comment:
Might be a controversial opinion, but I think for the "force" flag to be useful we should run every step of the `checkNoModelDependents` method, but log each violation instead of returning an error.

Reply:
Sure, I can invert it.